robozonky-installer/robozonky-installer-panels/src/main/java/com/github/triceo/robozonky/installer/panels/RoboZonkyInstallerListener.java

```diff
@@ -300,10 +311,36 @@ CommandLinePart prepareLogging() {
         }
     }
 
+    void prepareDaemons() {
+        Collection<String> contentC;
+        String content;
+        try {
+            SERVICES_PATH.mkdir();
+
+            content = Files.lines(Paths.get(DIST_SERVICES_PATH + "/robozonky-systemd.service")).collect(Collectors.joining("\n"));
+            content = content.replaceAll("/usr/robozonky", INSTALL_PATH.getAbsolutePath());
+            content = content.replaceAll("robozonky.sh @robozonky.cli", "run.sh");
+            contentC = new ArrayList<>();
```

Comment:
Potřebujete-li kolekci, ve které bude vždy jen jeden záznam, dá se to napsat jednodušeji: `Collections.singleton(content)` nebo `Collections.singletonList(content)` - podle toho, jestli potřebujete `Set` nebo `List`.